torch/onnx/_internal/fx/op_validation.py

```diff
@@ -76,10 +78,10 @@ def validate_op_between_ort_torch(
         )
     except ValueError as value_error:
         diagnostic = diagnostic_context.inflight_diagnostic()
-        diagnostic.with_additional_message(
-            f"### Op level debug fails due to unsupported input types\n"
-            f"{diagnostics.decorator.format_exception_in_markdown(value_error)}"
-        )
+        with diagnostic.log_section(
+            logging.WARNING, "Op level debug fails due to unsupported input types"
+        ):
+            diagnostic.log_source_exception(logging.WARNING, value_error)
```

Comment:
.warning is not used here, because we need `value_error`?

Reply:
`logging.WARNING` is passed as argument.